### DISCUSSION TITLE
[Feature request] button to check / uncheck all cores

### DISCUSSION THREAD

**Discussion by avrel-kb** [RetroDriven/Pocket_Updater#13] [Implemented Ideas]
Hi, thanks for you work it is very appreciated.

As the list of core keeps growing, I would really like to have a button to uncheck all selected cores in order to be able to choose just 1 or 2 without having to manually unselect everything.
There could also be a check all cores (but this would be less essential)

Avrel

**Comment by RetroDriven** [comment]
This has just been added with the new v1.2.0 Release.

https://github.com/RetroDriven/Pocket_Updater/releases/tag/v1.2.0

**Reply by avrel-kb** [reply]
Thanks a lot. It was quick.
It tried and it works well.

Now just a nit-picking comment (but really low priority).
Since when opening the "Manage Cores" menu, all cores are selected it would be more consistent to have "Select All" also selected.

**Reply by RetroDriven** [reply]
Good catch. I'll add that in for the next release.
